packages/react/slot/src/Slot.tsx

```diff
@@ -11,12 +11,11 @@ interface SlotProps extends React.HTMLAttributes<HTMLElement> {
 
 const Slot = React.forwardRef<HTMLElement, SlotProps>((props, forwardedRef) => {
   const { children, ...slotProps } = props;
-  const childArray = React.Children.toArray(children);
 
-  if (childArray.some(isSlottable)) {
+  if (React.Children.toArray(children).some(isSlottable)) {
     return (
       <>
-        {childArray.map((child) => {
+        {React.Children.map(children, (child) => {
```

Comment:
TIL!

I didn't expect this would fix the issue, without setting `key`, is `React.Children.map` doing something I am unaware of?

I assumed it was equivalent to `React.Children.toArray(children).map`.

Reply:
React will always use the index as the key if one isn't provided, but it warns when one isn't provided because providing one is the preferred approach. Using `React.Children.map` just suppresses the warning AFAIK.

Reply:
Awesome, TIL 🙂